core/src/css/component/replays/replay-info/replay-info-mercenaries.component.scss

```diff
@@ -110,5 +112,7 @@
 	.player-name {
 		margin-left: 15px;
 		width: auto;
+		min-width: 50px;
```

Comment:
Maybe even force the width to 50-80px, and truncate the name if it's longer than that? So that you're sure that everything stays aligned, even with long names.

Reply:
I don't think that misalignment is such a big problem that it would be worth cutting off nicknames. Today, everything is aligned in most cases, it seems to me that this is enough.

Reply:
Ok, I'll make it bigger if the need arises, I think there is enough room on the screen for that